about.html

```diff
@@ -32,17 +32,17 @@ <h2>About VuFind®</h2>
       support when you need it.
     </p>
 
-    <h2>Power</h2>
-    VuFind® runs on Solr Energy. Apache Solr, an open source search engine, offers amazing performance and scalability to allow for VuFind® to respond to search queries in milliseconds time. It has the ability to be distributed if you need to spread the load of the catalog over many servers or in a server farm environment.
+    <h2>Power and Flexibility</h2>
```

Comment:
Suggest as heading: _Powerful and Flexible Indexing_

Reply:
I adjusted the about headings in 0c1e2ababccbc7b0e2a8d5d798bf6a2034d4bd0c -- I used "Search Capabilities" instead of "Indexing" since I also want to account for scenarios where no local index is used.